.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "docs/themes/hugo-book"]
+	path = docs/themes/hugo-book
+	url = ../../alex-shpak/hugo-book
```

Comment:
Any reason not to use `github.com/alex-shpak/hugo-book`?

Reply:
Probably something my git client optimized, switched to full URL...